### ISSUE TITLE
INCLUDE from url/http?

### ISSUE THREAD

**Issue by Roemer** [devthefuture-org/dockerfile-x#5]
I think the current implementation only allows including from files, how about also allow including from an URL? Then the partial files could reside in a git repository or artifactory or such.

**Comment by devthejo** [comment]
Yes, as indicated on the README it's in the roadmap, see https://github.com/devthefuture-org/dockerfile-x#todo-next
But it's a lot of work and I don't have so much time for now.
If you want to try a PR I'll be happy to help you with,
instead you'll have to wait some months for this feature ;-)

**Comment by arthurmartelli** [comment]
Is this still on progress?

**Comment by devthejo** [comment]
@Roemer @arthurmartelli — shipped in **v1.6.0** 🎉

`dockerfile-x` now supports three remote include schemes for `INCLUDE`,
`INCLUDE_ARGS|ENVS|LABELS`, `FROM`, and `COPY/ADD --from=`:

- **HTTP(S)**
  ```Dockerfile
  INCLUDE https://example.com/snippets/setup.dockerfile
